Windows-Installation_Instructions.txt

```diff
@@ -0,0 +1,51 @@
+INSTALLATION OF SIMPHONY, SIMPHONY-NCAD AND NFLUID
+==================================================
+
+1. Install Python 2.7
+   - https://www.python.org/downloads/
+     (Windows package python-2.7.11.msi)
+
+2. Configure Python to use MinGW
+   - create 'distutils.cfg' file under 'C:\Python27\Lib\distutils' folder with the text:
+[build]
+compiler = mingw32
+
+3. Install Git
+   - http://www.git-scm.com/downloads/
+     (Windows package Git-2.7.0-64-bit)
+
+4. Copy 'MakeWin.bat' script to some folder 'FOO' (e.g. your Desktop)
+
+5. Open up a Git CMD terminal and change to that folder
+   - run 'git-cmd.exe'
+   - 'cd FOO'
+
+6. Execute the script 'MakeWin.bat' to install Python packages
+   - install pip package: 'MakeWin pip'
+   - install numpy package: 'MakeWin numpy'; if problems arise here then
+      . download numpy package from http://www.lfd.uci.edu/~gohlke/pythonlibs/#numpy to 'C:\Python27\Lib\site-packages'
+      . current package version is 'numpy-1.10.4+vanilla-cp27-none-win32.whl' (26/02/2016)
+      . install it manually by running: 'pip install C:\Python27\Lib\site-packages\numpy-1.10.4+vanilla-cp27-none-win32.whl'
```

Comment:
Great. The problem is that Christoph Gohlke's webpage, which provides the precompiled `whl` file for `numpy`, is not very _friendly_ to automatic downloads from the Windows command prompt. I try to bypass this limitation by setting the `User-Agent` into the `urllib2` request. It should work, but...

No problem if a human loads up the page and click on the specific link to download.